From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Grigoris Pavlakis <grigpavl@ece.auth.gr>
Date: Mon, 13 Jun 2022 16:04:43 +0300
Subject: [PATCH] Build: Disable nebu using SDL's glext.h constants

SerenityOS provides glext.h definitions inside GL/gl.h, but the
build process thinks that glext.h doesn't exist, therefore it attempts
to use SDL's definitions, which leads to a conflict. Therefore, disable
use of said definitions.
---
 nebu/include/video/nebu_renderer_gl.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/nebu/include/video/nebu_renderer_gl.h b/nebu/include/video/nebu_renderer_gl.h
index 64bf0a4..b74e3db 100644
--- a/nebu/include/video/nebu_renderer_gl.h
+++ b/nebu/include/video/nebu_renderer_gl.h
@@ -1,6 +1,8 @@
 #ifndef NEBU_RENDERER_GL_H
 #define NEBU_RENDERER_GL_H
 
+#define NO_SDL_GLEXT
+
 #include "SDL_opengl.h"
 #include <stdio.h>
 #include "video/nebu_quad.h"
